util/src/main/java/io/kubernetes/client/informer/cache/Store.java

```diff
@@ -0,0 +1,68 @@
+package io.kubernetes.client.informer.cache;
+
+import java.util.List;
+
+public interface Store<ApiType> {
```

Comment:
Isn't this basically just a Map?

Reply:
mmm.. let me try if it can be replaced by `Map`..

Reply:
ur point is making sense to me. i'm trying to make the `Store` interface inherit from `Map`. but this really brings a large refactor throughout the pull. 👀i'm not even confident if it will work the same as the verified one in our internal environment. wdyt if i add a TODO comment and refactor this in a follow-up pull?

Reply:
TODO is fine with me.

Reply:
added, thx